steps/install.sh

```diff
@@ -132,14 +95,7 @@ done < <( cut -d' ' -f2 "${root}/../DEPENDS.txt" | uniq )
 $SUDO tlmgr --verify-repo=none install "${packages[@]}"
 $SUDO tlmgr --verify-repo=none update --no-auto-remove "${packages[@]}" || echo 'Failed to update'
 
-if ! pygmentize -V >/dev/null 2>&1; then
-  if [ -n "${linux}" ]; then
-    apt-get install -y python3-pygments
-  else
-    echo "Install 'python3-pygments' somehow"
-    exit 1
-  fi
-fi
+install_or_quit 'python3-pygments' 
```

Comment:
@howcanunot there is no `python3-pygments` in MacOS when `pygmentize` is installed